src/components/sidebar/components/sidebar-nav/sidebar-nav-menu-item/index.tsx

```diff
@@ -3,6 +3,8 @@ import { MenuItem } from 'components/sidebar'
 import useCurrentPath from 'hooks/use-current-path'
```

Comment:
My sidebar PR implements external link icons, the Link component for internal links, dividers, and updated styles. Can we undo changes related to those item here?